### PR TITLE
Sp2 view profile reviews

### PR DESCRIPTION
**Pull request by parkaransingh**
Feature: 
Sprint2. Endpoint /getReviewsUser is reached in networking service and the user profile model includes the reviews generated by an email id.

Pair programmers: Parkaranjot Singh and Alexander Goebel and Madhura gangal

### CODE REVIEW

**Review by Spellbreeze — APPROVED** [review]

File: Civility Optics/ViewModels/UserProfileModel.swift [hunk 1]
```diff
@@ -12,8 +12,7 @@ class UserProfileModel: ObservableObject {
         email: String
     ) {
       self.email = email
-        self.post = Post(user: User())
-
```
Comment (Spellbreeze) [inline Civility Optics/ViewModels/UserProfileModel.swift:16]: Thank you for correcting spacing.

File: Civility Optics/ViewModels/UserProfileModel.swift [hunk 2]
```diff
@@ -29,9 +28,24 @@ class UserProfileModel: ObservableObject {
...
+        }
+        
     }
     
+    
+//    func getUserReviews() {
+//     print("started")
+//    }
+    
+
```
Comment (madhura112) [inline Civility Optics/ViewModels/UserProfileModel.swift:46]: Can get rid of unnecessary comments

File: Civility Optics/ViewModels/UserProfileModel.swift [hunk 3]
```diff
@@ -29,9 +28,24 @@ class UserProfileModel: ObservableObject {
...
+            DispatchQueue.main.async {
+              self.results = results
+            }
+          }
+        }
+        
     }
     
+    
+//    func getUserReviews() {
```
Comment (Spellbreeze) [inline Civility Optics/ViewModels/UserProfileModel.swift:42]: Remove unnecessary comment

File: Civility Optics/Views/Onboarding/RegisterView.swift [hunk 4]
```diff
@@ -229,8 +229,8 @@ struct RegisterView: View {
...
-struct RegisterView_Previews: PreviewProvider {
-    static var previews: some View {
-        RegisterView()
-    }
-}
+//struct RegisterView_Previews: PreviewProvider {
+//    static var previews: some View {
+//        RegisterView()
+//    }
+//}
```
Comment (madhura112) [inline Civility Optics/Views/Onboarding/RegisterView.swift:236]: Can remove this

File: Civility Optics/Views/Onboarding/profile.swift [hunk 5]
```diff
@@ -87,15 +87,15 @@ struct profile: View {
...
+        print(model.email)
       }
     }
-    struct profile_Previews: PreviewProvider {
-
-            static var previews: some View {
-                profile(model: UserProfileModel(email:""))
-        }
-    }
+//    struct profile_Previews: PreviewProvider {
```
Comment (Spellbreeze) [inline Civility Optics/Views/Onboarding/profile.swift:93]: Remove unnecessary comment

File: Civility Optics/Views/Onboarding/profile.swift [hunk 6]
```diff
@@ -87,15 +87,15 @@ struct profile: View {
...
+//
+//            static var previews: some View {
+//                profile(model: UserProfileModel(email:""))
+//        }
+//    }
```
Comment (madhura112) [inline Civility Optics/Views/Onboarding/profile.swift:98]: Can remove this

File: Civility Optics/Views/RateView.swift [hunk 7]
```diff
@@ -85,6 +84,10 @@ struct RateView: View {
       .accentColor(.velvet)
       .padding()
     }.navigationTitle("New Rating")
+          .onAppear {
```
Comment (Spellbreeze) [inline Civility Optics/Views/RateView.swift:87]: Fix spacing

File: Civility Optics/Views/SearchView.swift [hunk 8]
```diff
@@ -10,21 +10,13 @@ import SwiftUI
...
   var body: some View {
     results.navigationBarTitle("Civility Optics")
           .navigationBarBackButtonHidden(true)
-      
-//      TabView {
-//          SearchView(model: .init()).tabItem {
-//                    Label("Search Places", systemImage: "list.dash")
-//                          }
-//
-//                  }
```
Comment (madhura112) [inline Civility Optics/Views/SearchView.swift:25]: Don't need this

File: Civility Optics/Views/SearchView.swift [hunk 9]
```diff
@@ -63,17 +55,19 @@ struct SearchView: View {
   }
 }
 
-struct SearchView_Previews: PreviewProvider {
-  static var previews: some View {
-    NavigationView {
-      SearchView(model: SearchViewModel())
-    }
+
```
Comment (Spellbreeze) [inline Civility Optics/Views/SearchView.swift:58]: Remove unnecessary comment

**Review by madhura112 — APPROVED** [review]
Looks good, maybe need some minor formatting changes

**Review by Spellbreeze — APPROVED** [review]

**Pull request merged**